src/App.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+//import Organisation from'./OrganisationComp';
+
+
+
+class App extends Component {
+  renderServiceProviders = () => {
+    if (!this.state.orgData)
+        return <p> Data Not Displayed</p>;
+    let serviceProviders = [];
+    let organisationList = this.state.orgData.data;
+    //loops through list
+    for(let i = 0; i < organisationList.length; i++) {
+      serviceProviders.push(
+        //organisation component here!!
+        //<Organisation/>
+        <div>
+        <div className="list" key={i}>
+        <strong>Organisation: </strong>{organisationList[i].organisation}
+        <br></br> 
```

Comment:
You can just use <br /> although it is better to use CSS to add padding to your lines